.happy/terraform/envs/rdev/outputs.tf

```diff
@@ -0,0 +1,19 @@
+output frontend_url {
+  value       = module.stack.frontend_url
+  description = "The URL endpoint for the website service"
+}
+
+output backend_url {
+  value       = module.stack.backend_url
+  description = "The URL endpoint for the website service"
```

Comment:
same thing here (not sure how helpful that will actually be, i'm actually a little unclear on how these output variable descriptions will be used)

Reply:
The descriptions aren't really used, but the values are used by the Happy CLI to be able to find ECS task definitions and frontend/backend URL's.